.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+* @a11r @ctiller @nicolas-noble
+bazel/* @a11r @ctiller @nicolas-noble
+cmake/* @a11r @ctiller @jtattermusch @nicolas-noble
+doc/PROTOCOL-HTTP2.md @a11r @ctiller @ejona86 @nicolas-noble
+doc/interop-test-descriptions.md @a11r @ctiller @ejona86 @nicolas-noble
+etc/* @a11r @ctiller @jboeuf @nicolas-noble
+include/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/core/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/cpp/* @a11r @ctiller @dgquintas @markroth @nicolas-noble
+src/csharp/* @a11r @apolcyn @ctiller @jtattermusch @nicolas-noble
```

Comment:
There's also the option of doing a set noparent in the OWNERS files of src/... which we can play with in the case that top-level owners start getting too many review requests.